docs/databases/mysql/how-to-create-and-use-mysql-stored-procedures/index.md

```diff
@@ -0,0 +1,322 @@
+
+## Prepare the Database
+
+You will start by creating a sample database, table, and user for accessing the database. You will also populate the table with sample data for testing purposes.
+
+### Creating the Database, Table, and User
+
+1. Log into the MySQL server:
+
+        mysql -u root -p
+
+    You will be prompted to enter the root password of your MySQL database. Then, hit **Enter** to continue.
+
+1.  Next, you will see a MySQL prompt similar to the one shown below.
+
+    {{< output >}}
+mysql >
+{{< /output >}}
+
+1.  Enter the command below to create a `test_db` database:
+
+        Create database test_db;
+
+    Output:
+
+    {{< output >}}
+Query OK, 1 row affected (0.01 sec)
+{{< /output >}}
+
+1.  Create a database user and grant full access to the `test_db` database. Replace `PASSWORD` with a complex and unique value that follows the [guidelines for MySQL passwords](https://dev.mysql.com/doc/mysql-security-excerpt/5.6/en/user-names.html):
+
+        Create User 'test_user'@'localhost' IDENTIFIED BY 'PASSWORD';
+
+    Output:
+
+    {{< output >}}
+Query OK, 1 row affected (0.01 sec)
+{{< /output >}}
+
+1.  Grant the `test_user` full privileges to the `test_db` database;
+
+        GRANT ALL PRIVILEGES ON test_db.* TO 'test_user'@'localhost';
+
+    Output:
+
+    {{< output >}}
+Query OK, 1 row affected (0.01 sec)
+{{< /output >}}
+
+1.  Flush privileges:
+
+        FLUSH PRIVILEGES;
+
+    Output:
+
+    {{< output >}}
+Query OK, 0 rows affected (0.01 sec)
+{{< /output >}}
+
+### Populate the Database
+
+1.  Next, switch to the test_db database:
+
+        Use test_db;
+
+    Output:
+
+    {{< output >}}
+Database changed
+{{< /output >}}
+
+1.  Create a `products` table to store sample records:
+
+        Create table products(product_id BIGINT PRIMARY KEY, product_name VARCHAR(50), category_name VARCHAR(50) ) ENGINE=INNODB;
+
+    Output:
+
+    {{< output >}}
+Query OK, 0 rows affected (0.01 sec)
+{{< /output >}}
+
+1.  You can now add a few products to the products table by executing the commands below one by one:
+
+        Insert into products (product_id, product_name, category_name) values ('1', 'GAMING KEYBOARD', 'COMPUTER ACCESSORIES');
+        Insert into products (product_id, product_name, category_name) values ('2', 'OPTICAL MOUSE', 'COMPUTER ACCESSORIES');
+        Insert into products (product_id, product_name, category_name) values ('3', 'MOUSE PAD', 'COMPUTER ACCESSORIES');
+        Insert into products (product_id, product_name, category_name) values ('4', 'STEREO SYSTEM', 'ELECTRONICS');
+        Insert into products (product_id, product_name, category_name) values ('5', '32 INCH TV', 'ELECTRONICS');
+        Insert into products (product_id, product_name, category_name) values ('6', 'DVB-T2 RECEIVER', 'ELECTRONICS');
+
+    You should get the below output after executing each `Insert` statement.
+Select * from products where category_name=category;

```

Comment:
This should be indented to make a code block

Reply:
This code was actually an accidental copy-paste that didn't need to be there, so I removed it